hooks/class.tx_dlf_em.php

```diff
@@ -59,13 +59,14 @@ public function checkSolrConnection(&$params, &$pObj) {
         }
 
         // Set port if not set.
-        $port = (!empty($this->conf['solrPort']) ? \TYPO3\CMS\Core\Utility\MathUtility::forceIntegerInRange($this->conf['solrPort'], 0, 65535, 8180) : 8180);
+        $port = (!empty($this->conf['solrPort']) ? \TYPO3\CMS\Core\Utility\MathUtility::forceIntegerInRange($this->conf['solrPort'], 0, 65535, 8983) : 8180);
```

Comment:
We should use `tx_dlf_solr::getSolrConnectionInfo()` here instead of building the base URL from configuration again.
(I am not sure if this works in the context of the extension manager, though.)